### PR TITLE
1219 aggs for mutations count

### PR DESCRIPTION
**Pull request by cy**
needs https://github.com/NCI-GDC/gdcapi/pull/799 and https://github.com/NCI-GDC/gdcapi/pull/798

### CODE REVIEW

**Comment by cy** [comment]
this is ready for review, needs NCI-GDC/gdcapi#799 to work, NCI-GDC/gdcapi#798 for the counts to be correct

File: src/packages/@ncigdc/containers/CancerDistributionTable.js [hunk 1]
```diff
@@ -187,8 +209,7 @@ const CancerDistributionTableComponent = compose(
             ? {
                 num_mutations: (
                   <MutationsCount
-                    key={row.project_id}
-                    ssms={explore.ssms}
+                    ssmCount={ssmCounts[row.project_id]}
```
Comment (Jephuff) [inline src/packages/@ncigdc/containers/CancerDistributionTable.js]: Could you add this to the `rawData` so it's included in the json download?

Comment (cy) [inline src/packages/@ncigdc/containers/CancerDistributionTable.js]: ohhh yup thanks for the catch

Comment (cy) [inline src/packages/@ncigdc/containers/CancerDistributionTable.js]: done

**Pull request merged**